cassandra-mesos-model/src/main/proto/io/mesosphere/mesos/frameworks/cassandra/model.proto

```diff
@@ -74,7 +109,11 @@ message CassandraNode {
     optional CassandraNodeTask metadataTask = 6;
```

Comment:
No - I think we should keep it there because we should keep it ”constant” over executor restarts. Gets a bit more important if there's JMX authentication in place ... sometimes.